activerecord/lib/active_record/relation.rb

```diff
@@ -554,10 +554,12 @@ def where_values_hash(relation_table_name = klass.table_name)
       where_clause.to_h(relation_table_name)
     end
 
-    def scope_for_create(attributes = nil)
-      scope = where_values_hash.merge!(create_with_value.stringify_keys)
-      scope.merge!(attributes) if attributes
-      scope
+    def scope_for_create
+      where_values_hash.merge!(create_with_value.stringify_keys)
+    end
+
+    def values_for_create(attributes = nil)
```

Comment:
Add `# :nodoc:`

Reply:
sure, should I make this method private then?

Reply:
Yeah, that would be great.